includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -132,7 +132,7 @@ public static function add_admin_hooks() {
 
 		// Hide irrelevant "published" label in the invalid URL post list.
 		add_filter( 'post_date_column_status', function( $status, $post ) {
-			if ( self::POST_TYPE_SLUG === get_post_type( $post ) ) {
+			if ( AMP_Invalid_URL_Post_Type::POST_TYPE_SLUG === get_post_type( $post ) ) {
```

Comment:
It's great that we're going through this exercise of creating these tests now since otherwise this PHP 5.3 issue would have slipped through.